src/test/java/seedu/sellsavvy/logic/commands/ordercommands/OrderCommandTestUtil.java

```diff
@@ -118,4 +120,18 @@ public static void showOrderAtIndex(Model model, Index targetIndex) {
 
         assertEquals(1, model.getFilteredPersonList().size());
     }
+
+    /**
+     * Returns the order in the model's selected filtered order list by the given {@code Index}.
+     */
+    public static Order getOrderByIndex(Model model, Index index) {
+        return model.getFilteredOrderList().get(index.getZeroBased());
+    }
+
+    /**
+     * Returns the order list in the model's selected person's filtered order list by the given {@code Index}.
```

Comment:
order list in selected person's filtered order list?
How can an order list be in another order list?

Reply:
```suggestion
     * Returns an {@code OrderList} of the model's selected person by the given {@code Index}.
```